app/frontend/src/features/communities/CommunityPage/EventCard.tsx

```diff
@@ -54,12 +57,19 @@ export default function EventCard({
   const classes = useStyles();
   const date = timestamp2Date(event.startTime);
   return (
-    <Card className={classNames(classes.root, className)}>
-      <CardActionArea>
+    <Card className={className}>
+      <Link
+        to={routeToEvent(event.eventId ?? 0, event.slug ?? "")}
+        className={classes.link}
+        component={CardActionArea}
+      >
         <CardContent>
```

Comment:
```suggestion
      >
      <CardActionArea>
        <CardContent>
        ... some other stuff
      </CardActionArea>
```
Doing that seems to work, will have to fix the link styling to not get the horrible default blue text.